metadata-ingestion/src/datahub/ingestion/source/usage/clickhouse_usage.py

```diff
@@ -150,9 +151,12 @@ def _get_clickhouse_history(self):
                 if isinstance(v, str):
                     event_dict[k] = v.strip()
 
-            if not self.config.schema_pattern.allowed(
-                event_dict.get("schema_")
-            ) or not self.config.table_pattern.allowed(event_dict.get("table")):
+            if (
+                    not self.config.database_pattern.allowed(event_dict.get("database")) or
+                    not self.config.table_pattern.allowed(event_dict.get("full_table_name")) or
+                    not self.config.view_pattern.allowed(event_dict.get("full_table_name"))
```

Comment:
we only want to skip if it is not allowed by both the table pattern and the view pattern
```suggestion
                    not (self.config.table_pattern.allowed(event_dict.get("full_table_name")) or
                   self.config.view_pattern.allowed(event_dict.get("full_table_name")))
```